CIScripts/Build/BuildFunctions.ps1

```diff
@@ -196,7 +197,10 @@ function Invoke-ExtensionBuild {
         Copy-Item $utilsMSI $OutputPath -Recurse -Container
         Copy-Item $vRouterMSI $OutputPath -Recurse -Container
         Copy-Item $vRouterCert $OutputPath -Recurse -Container
-        Copy-Item $vTestPath "$OutputPath\utils\vtest" -Recurse -Container
+        New-Item -Type Directory -Path "$OutputPath\utils\vtest" | Out-Null
+        Copy-Item "$vTestSrcPath\tests" "$OutputPath\utils\vtest" -Recurse -Filter "*.xml"
+        Copy-Item "$vTestSrcPath\*.ps1" "$OutputPath\utils\vtest"
+        Copy-Item "$vTestPath\*" "$OutputPath\utils\vtest" -Recurse -Container
```

Comment:
Maybe moving vtest and related xmls and scripts should be moved to a different Job.Step?